### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by Charlottecuc** [yl4579/AuxiliaryASR#4]
Hi. Could you kindly share your training loss of the model (maybe a tensorboard picture)? Thank you very much.

**Comment by yl4579** [comment]
![image](https://user-images.githubusercontent.com/71044569/173625614-53fb8da7-8041-493b-8d9a-615e4b087a10.png)

**Comment by hai8023** [comment]
Why are there negative values? Shouldn't loss be positive in general?
![WeChat Image_20220615150238](https://user-images.githubusercontent.com/83195382/173763596-ff9c8f1f-32d6-4865-a4e2-988ae8fafd2b.png)

**Comment by yl4579** [comment]
@hai8023 This happens when a lot of blank tokens appear in the targets for CTC loss. You need to check your labels to make sure it doesn't a space (` `). 
However, even if you don't have it, the optimal CTC loss will be negative because the blank token was added to the beginning and the end of the speech to indicate silence before and after the speech. This is intended behavior unless your audio is trimmed to have no silence at the beginning and the end of the speech. For more explanation, please see https://discuss.pytorch.org/t/negative-ctc-loss/79548/5

**Comment by hai8023** [comment]
Thank you very much for your reply
